target/mips/helper.h

```diff
@@ -213,6 +213,7 @@ DEF_HELPER_3(cbts, tl, env, i32, i32)
 DEF_HELPER_3(cbtu, tl, env, i32, i32)
 DEF_HELPER_3(ccall, void, env, i32, i32)
 DEF_HELPER_3(ccall_notrap, tl, env, i32, i32)
+DEF_HELPER_3(ccall_notrap2, tl, env, i32, i32)
```

Comment:
Would be nice to give this a better name than CCall(NoTrap)2. CCallU or something?

Reply:
I agree a better name would be nice. U sort of implies it only works on unsealed things, maybe something conditional in the name? CCCall is a silly name though...

Reply:
Keeping as is for now.